### ISSUE TITLE
Unable to access interface: Operation not permitted (pivpn)

### ISSUE THREAD

**Issue by KevHH** [donaldzou/wireguard-dashboard#30]
**Describe The Problem**
Thanks for the amazing tool! This issue may be related to #8 but I am not exactly sure as I am quite new to wireguard and pivpn.

I am able to see the dashboard for the configuration wg0 I created and the peers under them. However when a peer connects, there's no change in the data. The log shows "unable to access interface: Operation not permitted" error.

**Expected Error / Traceback**
```
127.0.0.1 - - [11/Jul/2021 02:06:58] "GET /configuration/wg0 HTTP/1.1" 200 -
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
127.0.0.1 - - [11/Jul/2021 02:06:58] "GET /get_config/wg0 HTTP/1.1" 200 -
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
127.0.0.1 - - [11/Jul/2021 02:06:58] "GET /get_config/wg0 HTTP/1.1" 200 -
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
127.0.0.1 - - [11/Jul/2021 02:06:58] "GET /get_config/wg0 HTTP/1.1" 200 -
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
Unable to access interface: Operation not permitted
```

**To Reproduce**
Basically following the instructions and running `./wgd.sh start`. I have done `chmod` for `/etc/wireguard` as instructed in README too.

**OS Information:**
 - OS: Raspbian GNU/Linux 10 (buster)
 - Python Version: v3.7.3

**Sample of your `.conf` file**
```
[Interface]
PrivateKey = ***
Address = 10.6.0.1/24
MTU = 1420
ListenPort = 51820
SaveConfig = true
### begin iPhone ###
[Peer]
PublicKey = ***
PresharedKey = ***
AllowedIPs = 10.6.0.2/32
### end iPhone ###
### begin Windows ###
[Peer]
PublicKey = ***
PresharedKey = ***
AllowedIPs = 10.6.0.3/32
### end Windows ###
```

**Comment by donaldzou** [comment]
Hi @KevHH ! Could you try to use `sudo ./wgd debug` and see if you run it? Because I've noticed that `pivpn` will not give the regular `pi` account the permission to run `wg show...` or `wg-quick`

**Comment by KevHH** [comment]
Thanks that's super helpful! I fixed it by running `sudo python3 -m pip install -r requirements.txt` followed by `sudo ./wgd.sh start`. Doing the second command alone would raise errors about some missing packages.

**Comment by tandy-1000** [comment]
I'm having this issue (not with PiVPN) but I'm not sure whether it's safe to run wgd.sh as root?
